htdocs/admin/receiptprinter.php

```diff
@@ -43,7 +43,7 @@
 $printerid = GETPOST('printerid', 'int');
 $parameter = GETPOST('parameter', 'alpha');
 
-$template = GETPOST('template', 'alpha');
+$template = GETPOST('template', 'none');
```

Comment:
May be you can use "restricthtml" instead of "none" ?
What is format of template ?